src/test/components.js

```diff
@@ -92,13 +92,21 @@ const XYPLOT_SERIES_PROPS = {
 const XYPLOT_XAXIS_PROPS = {
   xRange: [0, 1],
   xDomain: [0, 1],
-  xType: 'linear'
+  xType: 'linear',
+  width: 100,
+  height: 100,
+  top: 0,
+  left: 0
```

Comment:
We do not allow the user to work with the margins anymore. If the user wants to align the elements inside the XYPlot, he should use top, left, width and height. Otherwise, those values will be picked up by the margins.

`marginLeft`, `marginRight`, `marginTop`, `marginBottom`, `innerWidth` and `innerHeight` properties are still passed by XYPlot to set the default position